attributes/default.rb

```diff
@@ -31,3 +31,6 @@
 default['activemq']['enable_stomp'] = true
 default['activemq']['use_default_config'] = false
 default['activemq']['install_java'] = true
+# set to 'transport.enabledProtocols=TLSv1,TLSv1.1,TLSv1.2'
+# to disable sslv3 and protect against poodle
+default['activemq']['transport_protocols'] = nil
```

Comment:
It is, but I am including the flag here for completeness and self documentation.